src/khoj/processor/text_to_jsonl.py

```diff
@@ -12,14 +12,14 @@
 from khoj.utils.rawconfig import Entry
```

Comment:
All these `x_to_jsonl` files also need to be renamed to something like `x_to_entries` or `x_to_embeddings`. Do you want to take care of that as part of this PR as well?

Reply:
Good idea. Updated `s/x_to_jsonl/x_to_entries` for files and `s/XToJsonl/XToEntries/` for class names